examples/01_plotting/plot_surf_atlas.py

```diff
@@ -77,6 +77,34 @@
                        darkness=.5)
 plotting.show()
 
+###############################################################################
+# Display connectome from surface parcellation
```

Comment:
A little more text explaining what you are doing here would be useful for non experts